components/engine/vendor/github.com/containerd/containerd/runtime/container_linux.go

```diff
@@ -176,7 +177,7 @@ func (c *container) getMemoryEventFD(root string) (*oom, error) {
 		return nil, err
 	}
 	defer f.Close()
-	fd, _, serr := syscall.RawSyscall(syscall.SYS_EVENTFD2, 0, syscall.FD_CLOEXEC, 0)
+	fd, _, serr := syscall.RawSyscall(syscall.SYS_EVENTFD2, 0, unix.EFD_CLOEXEC, 0)
```

Comment:
(OT) Is this fix included in containerd 1.0 as well? The code changed, so this file is no longer there, so just double-checking

Reply:
It moved to the containerd/cgroups repo and the current tip includes the fix

Reply:
Oh, that's a 0.2.x version :)